src/machine-learning/machine-learning.ts

```diff
@@ -131,7 +142,12 @@ export class MachineLearning implements FirebaseServiceInterface {
    * @return {Promise<Model>} A Promise fulfilled with the unpublished model.
    */
   public unpublishModel(modelId: string): Promise<Model> {
-    throw new Error('NotImplemented');
+    const updateMask = ['state.published'];
```

Comment:
This is almost identical to `publishModel` except for the value of `published` property. May be refactor into a helper function?

```
private setPublishStatus(modelId: string, published: boolean): Promise<Model>
```

Reply:
done